security-admin/src/main/java/org/apache/ranger/security/handler/RangerAuthenticationProvider.java

```diff
@@ -93,6 +94,13 @@ public Authentication authenticate(Authentication authentication)
 					return authentication;
 				}
 			}
+		}else if (isAlt_ssoEnabled()){
+			if(authentication != null){
```

Comment:
space after {, space after if, space after ) (or rather, keep it at least consistent with your own lines in this file.

} else if (isAlt_ssoEnabled()) {
    if (authentication != null) {